routing/turns_notification_manager.cpp

```diff
@@ -73,12 +73,13 @@ NotificationManager NotificationManager::CreateNotificationManagerForTesting(
 
 string NotificationManager::GenerateTurnText(uint32_t distanceUnits, uint8_t exitNum,
                                              bool useThenInsteadOfDistance, TurnItem const & turn,
-                                             measurement_utils::Units lengthUnits) const
+                                             measurement_utils::Units lengthUnits,
+                                             std::string const & nextStreet) const
```

Comment:
Maybe change to:
```cpp
std::string const & nextStreet = "") const
```
So you don't have to write `GenerateTurnText(..., "")` if there isn't a next street (see below)